### PR TITLE
Translation-ja: Add product-owner/ja/02-how-difficult-it-is-to-be-a-middle-manager

### PR DESCRIPTION
**Pull request by ystk**
This PR relates to #407 to add second article in Product Owner chapter.

### CODE REVIEW

File: product-owner/ja/02-how-difficult-it-is-to-be-a-middle-manager-article-ja.asciidoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+* 最初は、混沌と機能不全に陥ったプロダクトチームを引き継ぐことです。
+* 2番目は、柔軟性がなく、創造性を発揮する余地がほとんどないことです。多くの場合、これには明確な道筋が含まれていません。
+* 3番目、そして私が個人的に一番取り組まなければならないこととして、政治と内紛に対処することのストレスがあります。
+* 4番目は、中間管理職が仕事に対してほとんど信用を得られないことです。これは、単に業績だけではありません。上位の管理職は、次に何を行うかにだけフォーカスしており、実現不可能な期限を選んだり、目標を変更した事実を無視することが良くあります。したがって、完全に信用の問題になっているようです。
+* そして最後に5番目、ほとんどの人は真のリーダーになる自由を持つのではなく、記録係や執行官のように感じています。
+
+合っていますか？同意いただけますか？忘れている事があれば、下のコメントで知らせてください。
+
+こうした不満が、私がInnerSourceに取り組むことにした理由です。なぜなら中間管理職として、あなたの立場で数マイル以上歩いてきたからです。
```
Comment (johtani) [inline product-owner/ja/02-how-difficult-it-is-to-be-a-middle-manager-article-ja.asciidoc:25]:
```suggestion
こうした不満が、私がInnerSourceに取り組むことにした理由です。なぜなら中間管理職として、私も同じ立場だったからです。
```
a mileなどが入る慣用句があるみたいです。なので、こんな感じでどうでしょう？
https://keep-on-walking.net/walk-in-my-shoes/

**Review by johtani — COMMENTED** [review]
Great @ystk .
I left a suggestion.
Please apply if you are OK.

**Comment by ystk** [comment]
Great suggestion! I applied the suggested change. Thanks a lot @johtani !

**Review by johtani — APPROVED** [review]
LGTM!

**Comment by rrrutledge** [comment]
Excellent!

**Pull request merged**